config/suppressions.xml

```diff
@@ -14,6 +14,10 @@
               files="TokenTypes.java|IndentationCheckTest.java"
               lines="1"/>
 
+    <!--MainTest.java contains generated x-path which is very long-->
+    <suppress checks="FileLength|MethodLength"
+              files="MainTest.java"/>
```

Comment:
How this is related to update ? it is not ok, please provide detailed explanation.